mathml/elements/maction.json

```diff
@@ -7,8 +7,16 @@
           "spec_url": "https://w3c.github.io/mathml-core/#dfn-maction",
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "See <a href='https://crbug.com/6606'>bug 6606</a>."
+              "version_added": "105",
+              "impl_url": "https://crrev.com/c/3720717",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-experimental-web-platform-features",
+                  "value_to_set": "Enabled"
+                }
+              ],
+              "notes": "Only the basic implementation specified in MathML Core."
```

Comment:
Could we expand on this note?  Without more context, I have no idea what this means in practice, or how it impacts web developers.

Reply:
Yes please!  The notes should be as informative as possible so that a web developer will know exactly what to look out for when writing their website's code.  I think that sentence you just wrote would work perfectly for the notes!

Reply:
Got it. Done.